numpy/lib/tests/test_twodim_base.py

```diff
@@ -273,6 +276,27 @@ def test_binparameter_combination(self):
         assert_array_equal(H, answer)
         assert_array_equal(xe, array([0., 0.25, 0.5, 0.75, 1]))
 
+    @requires_array_function
+    def test_dispatch(self):
+        class ShouldDispatch:
```

Comment:
We probably need a new style class for Python 2:
```suggestion
        class ShouldDispatch(object):
```